### DISCUSSION TITLE
Inverse of HasRelated?

### DISCUSSION THREAD

**Discussion by nikhiltri** [area17/twill#886] [Ideas]
Hello,

It seems that the src/Models/Behaviors/HasRelated.php trait doesn't provide an inverse relationship to help identify things that have been related to a thing. In the trait could we add a relationship method like:

```
    public function relatedTo()
    {
        return $this->morphedByMany(RelatedItem::class, 'related');
    }
```

I'm happy to submit a pull request if that sounds ok.

Thanks!
nikhil

**Comment by ifox** [comment]
Hi Nikhil, a PR to add this would be nice! We are going to be working on a feature around providing visibility of inverse relationships between records in the UI very soon, including in the media library. So this will definitely be useful!
